do_cmake.sh

```diff
@@ -49,14 +51,19 @@ if type ccache > /dev/null 2>&1 ; then
     ARGS+=" -DWITH_CCACHE=ON"
 fi
 
+if [[ $CEPH_NO_DEBUG > 0 ]] ; then
+    echo "disabling debug build"
+    ARGS+=" -DCMAKE_BUILD_TYPE=RelWithDebInfo"
```

Comment:
@matthewoliver why not just launch `do_cmake.sh` like
```sh
./do_cmake.sh '-DCMAKE_BUILD_TYPE=RelWithDebInfo'
```
if you want to have a release build?

Reply:
Yeah that's true. and you still can. I was more concerned about the warnings the docs and script gives and wanted to make a full proof way and something that was very obvious.

That being said, I'm not married to the idea of CEPH_NO_DEBUG, as it seems weird to have an env variable with a negitive.. at least to me :)

Happy to remove that part of if it doesn't really make sense.

Reply:
I've removed the CEPH_NO_DEBUG part of the patch.